tests/test_libzim_reader.py

```diff
@@ -17,7 +17,7 @@
 ZIMS_DATA = {
     "blank.zim": {
         "filename": "blank.zim",
-        "filesize": 1173,
+        "filesize": 2197,
         "new_ns": True,
         "mutlipart": False,
```

Comment:
!? Multipart?!

Reply:
PRs are not the place to comment on code (test actually) not modified by such PR. We test the `is_multipart` method in tests and this above is a description of what to expect for this ZIM file (no multipart)